src/recoil/atom.tsx

```diff
@@ -0,0 +1,11 @@
+import { atom } from 'recoil';
+
+export const userState = atom({
+  key: 'userState', // unique ID (with respect to other atoms/selectors)
+  default: {
+    id: 811,
+    nickname: 'test',
+    email: 'test',
+    /*Workspaces: IWorkspace[]; // TODO 자신이 속한 워크스페이스 목록 -> 4주차때 확장*/
+  }, // default value (aka initial value)
+});
```

Comment:
```suggestion
import { atom } from 'recoil';
import userData from '../db/userData.json'

export const userState = atom({
  key: 'userState', // unique ID (with respect to other atoms/selectors)
  default: userData, // default value (aka initial value)
});

```

직접 작성하신 json data로 recoil의 default value를 set 해 주는 것이 좋을 것 같아요!